bin/gltf-pipeline.js

```diff
@@ -38,7 +39,7 @@ fs.readFile(gltfPath, function (err, data) {
         throw err;
     }
 
-    var gltf = JSON.parse(data);
+    var gltf = parseGLTF(path.dirname(gltfPath), data);
```

Comment:
To follow camel case naming conventions and to better show that this is to create our in-memory runtime data structure for glTF, let's call this `loadGltfUris`.

Let's also do the `JSON.parse` before calling the function, so the function should take a glTF object, not a string.